python/ray/worker.py

```diff
@@ -17,6 +17,7 @@
 
 # Ray modules
 from ray.autoscaler._private.constants import AUTOSCALER_EVENTS
+import ray._private.package as ray_pkg
```

Comment:
Import order is a bit odd looking here, I don't think we enforce, but prefer to keep them roughly sorted / grouped.